api/util/mappers/sdk.py

```diff
@@ -48,5 +48,5 @@ def map_environment_to_sdk_document(environment: "Environment") -> SDKDocument:
     ]
 
     return engine_environment.model_dump(
-        exclude=SDK_DOCUMENT_EXCLUDE,
+        exclude=SDK_DOCUMENT_EXCLUDE,  # type: ignore
```

Comment:
I'm not sure why we've added specific ignores here vs just letting the `mypy_baseline.txt` account for these errors?

Reply:
I was dealing with a bug. I've reverted the change and just added them to the baseline.